app/views/home/index.html.erb

```diff
@@ -27,3 +27,27 @@
   </div>
 </div>
 
+<div class="container">
+  <div class="row">
+    <div class = "col-lg-12 mt-5">
+      <h1> Browse recipes </h1>
+      <ul>
+        <%= will_paginate @recipes %>
+        <% @recipes.each do |recipe| %>
+          <%= recipe.id %>
+          <div class = "row">
```

Comment:
This should be an `li` tag as it's the first child inside a `ul` tag.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

Reply:
Oops, yes! Good point. I actually updated this to instead follow the semantic style html `<dl><dt><dd>` instead.